about.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: single-page-md
+title: About APIO 2020
+key: about
+---
+
+The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the Asian and Western Pacific regions. The first competition was organized by the Australian IOI team in 2007.
+
+APIO 2020 will be hosted by Indonesia. Usually, the contest will be held online.
```

Comment:
`s/Usually, the contest will be held online./In the past, APIO contests were held online.`